database/src/main/resources/db/migration/V60__revurdering.sql

```diff
@@ -0,0 +1,11 @@
+create table if not exists revurdering
+(
+    id uuid primary key,
+    opprettet timestamp with time zone not null,
+    behandlingId uuid references behandling(id) not null,
+    periode jsonb not null,
+    beregning jsonb,
+    simulering jsonb,
+    saksbehandler text,
```

Comment:
For nåværende impl., kan vel ikke denne være null?

Reply:
nei